common/src/main/templates/emails/incoming.mustache

```diff
@@ -3,7 +3,7 @@
         <div style="font-size: 24px">NEW WORKSPACE REQUEST</div>
     </div>
     <div style="width: 50%; padding: 25px; margin: 50px auto; background-color: #fff">
-        <p>A new workspace has been requested and is waiting for your review. Click below to view it.</p>
+        <p>A new workspace: {{workspaceName}} has been requested and is waiting for your review. Click below to view it.</p>
```

Comment:
nit: the colons are a little strange, how about just single quoting the workspace name `A new workspace '{{workspaceName}}' has been requested a...`